### PR TITLE
Ajax Manager

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: assets/dev/js/editor/components/saver/manager.js [hunk 1]
```diff
@@ -146,7 +146,11 @@ module.exports = Module.extend( {
 
 		self.isChangedDuringSave = false;
 
-		self.xhr = elementor.ajax.send( 'save_builder', {
+		if ( 'autosave' !== options.status && statusChanged ) {
+					elementor.settings.page.model.set( 'post_status', options.status );
```
Comment (KingYes) [inline assets/dev/js/editor/components/saver/manager.js]: tab tab

File: assets/dev/js/editor/elements/models/element.js [hunk 2]
```diff
@@ -164,11 +164,11 @@ ElementModel = Backbone.Model.extend( {
 	createRemoteRenderRequest: function() {
 		var data = this.toJSON();
 
-		return elementor.ajax.send( 'render_widget', {
+		return elementor.ajax.addRequest( 'render_widget', {
```
Comment (KingYes) [inline assets/dev/js/editor/elements/models/element.js]: send + via ajax-manager

File: assets/dev/js/editor/utils/ajax.js [hunk 3]
```diff
@@ -16,6 +18,125 @@ Ajax = {
...
+		var self = this;
+		if ( ! request.unique_id ) {
+			request.unique_id = request.action;
+		}
+
+		if ( request.before ) {
+			request.before();
+		}
+
+		var	deferred,
```
Comment (KingYes) [inline assets/dev/js/editor/utils/ajax.js]: tab

File: core/settings/page/manager.php [hunk 4]
```diff
@@ -5,6 +5,7 @@
 use Elementor\Core\Settings\Base\Manager as BaseManager;
 use Elementor\Core\Settings\Manager as SettingsManager;
 use Elementor\Core\Settings\Base\Model as BaseModel;
+use Elementor\Plugin;
```
Comment (KingYes) [inline core/settings/page/manager.php]: remove

**Review by KingYes — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/elements/models/element.js (hunk 2 above)
Comment (matipojo) [reply assets/dev/js/editor/elements/models/element.js]: done

**Pull request closed without merging**